blackboxopt/visualizations/utils.py

```diff
@@ -172,27 +175,6 @@ def add_plotly_buttons_for_logscale(fig):
     fig.update_layout(updatemenus=updatemenus)
 
 
-def mask_pareto_efficient(costs: np.ndarray):
-    """For a given array of objective values where lower values are considered better
-    and the dimensions are samples x objectives, return a mask that is `True` for all
-    pareto efficient values.
-
-    NOTE: The result marks multiple occurrences of the same point all as pareto
-    efficient.
-    """
-    is_efficient = np.ones(costs.shape[0], dtype=bool)
-    for i, c in enumerate(costs):
-        if not is_efficient[i]:
-            continue
-
-        # Keep any point with a lower cost or when they are the same
-        efficient = np.any(costs[is_efficient] < c, axis=1)
-        duplicates = np.all(costs[is_efficient] == c, axis=1)
-        is_efficient[is_efficient] = np.logical_or(efficient, duplicates)
-
-    return is_efficient
```

Comment:
This just moved to the general, visualization unspecific, utils (so did the test).